pkg/commands/kvstore/kvstore_test.go

```diff
@@ -71,6 +72,37 @@ func TestCreateStoreCommand(t *testing.T) {
 				UpdatedAt: &now,
 			}),
 		},
+		{
+			Args: testutil.Args(fmt.Sprintf("%s create --name %s --location %s", kvstore.RootName, storeName, storeLocation)),
+			API: mock.API{
+				CreateKVStoreFn: func(i *fastly.CreateKVStoreInput) (*fastly.KVStore, error) {
+					return &fastly.KVStore{
+						StoreID: storeID,
+						Name:    i.Name,
+					}, nil
+				},
+			},
+			WantOutput: fstfmt.Success("Created KV Store '%s' (%s)", storeName, storeID),
+		},
+		{
+			Args: testutil.Args(fmt.Sprintf("%s create --name %s --location %s --json", kvstore.RootName, storeName, storeLocation)),
+			API: mock.API{
+				CreateKVStoreFn: func(i *fastly.CreateKVStoreInput) (*fastly.KVStore, error) {
+					return &fastly.KVStore{
+						StoreID:   storeID,
+						Name:      i.Name,
+						CreatedAt: &now,
+						UpdatedAt: &now,
+					}, nil
+				},
+			},
+			WantOutput: fstfmt.EncodeJSON(&fastly.KVStore{
+				StoreID:   storeID,
+				Name:      storeName,
+				CreatedAt: &now,
+				UpdatedAt: &now,
+			}),
+		},
```

Comment:
I'm not sure of the value of the tests here other than they verify a `--location` flag can be provided.

It might be worth just adding a NOTE above the tests to indicate they just validate the flag is allowed as no other indicator is present in either the `fastly.KVStore` struct or subsequently the CLI output.

e.g.

```
// NOTE: The following tests only validate support for the --location flag.
// No other location/region indicators are exposed for us to validate.
```

Unless, you decide to change the CLI output to say something like...

```
fstfmt.Success("Created KV Store '%s' (%s, %s)", storeName, storeID, storeLocation)
```

...so we can validate the location.

Reply:
Thank you for your suggestion. IMHO, the API response should ideally include the location, and the CLI should use that value for the create command output.
(Furthermore, I think it would be preferable to include the location in the describe command output as well.)

Since the current API response does not include that value, I added a comment to the test as suggested.

I just noticed that the absence of location in the API response has been brought up internally. Hopefully, it will become available soon, and it can be incorporated into the CLI 🤞